CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Looks good, but can we please have it as a separate PR?

Reply:
Ah yes sorry, jumbled my local branches. It's fixed now.